Sources/PathKit.swift

```diff
@@ -608,6 +617,17 @@ extension Path : Sequence {
   public func makeIterator() -> DirectoryEnumerator {
     return DirectoryEnumerator(path: self)
   }
+
+  /// Perform a deep enumeration of a directory.
+  ///
+  /// - Parameter options: FileManager directory enumerator options.
+  ///
+  /// - Returns: a directory enumerator that can be used to perform a deep enumeration of the
+  ///   directory.
+  ///
+  public func makeIterator(options: FileManager.DirectoryEnumerationOptions) -> DirectoryEnumerator {
```

Comment:
This would unfortunately expose `FileManager` in the public API for PathKit. This would mean that the implementation is always tied to `FileManager`. I'd rather we didn't expose `FileManager` as apart of the API and instead provided our own enum to not expose implementation details to users of PathKit.